core/metrics-core-service/src/main/antlr4/org/hawkular/metrics/core/service/tags/parser/TagQuery.g4

```diff
@@ -67,6 +69,10 @@ array_operator
   | NOT IN
   ;
 
+existence_operator
+  : NOT
```

Comment:
So, is it acted that we won't use "exists" / "not exists" as existence operators, but instead just no operator for existence and just "not" for non-existence?
I'm asking because I'm implementing it in Grafana.